### ISSUE TITLE
Some integer is too big

### ISSUE THREAD

**Issue by Vistril** [KenanY/backpack.tf#9]
When i use .getUsers, I get this error.
(node:124) TimeoutOverflowWarning: 4294967296000 does not fit into a 32-bit signed integer.Timer duration was truncated to 2147483647.null

**Comment by kenany** [comment]
Yeah, this warning is coming from `hyperquest` (which `jsonist` uses). We probably just have to update `jsonist`.
